File: fizzbuzz.c
int main(){
  for(int i = 1; i <= 100; i++) {
    if(i % 3 == 0) {
      puts("Fizz\n");
    }

    else if(i % 5 == 0) {
      puts("Buzz\n");
    }

    else if(i % 3 == 0 && i % 5 == 0) {
      puts("FizzBuzz\n");
    }

    else {
      puts("%d\n", i);
    }
  }
}

File: fizzbuzz.java
public static void main(String[] args) {
    for(int i = 1; i <= 100; i++) {
        if(i % 3 == 0) {
            System.out.print("fizz\n");
        }
        
        else if(i % 5 == 0) {
            System.out.print("buzz\n");
        }

        else if(i % 3 == 0 && i % 5 == 0) {
            System.out.print("fizzbuzz\n");
        }

        else {
            System.out.print(i + "\n");
        }
    }
}